src/workloads/scale/MixedWorkloadsGennyRateLimited.yml

```diff
@@ -192,5 +192,7 @@ AutoRun:
       - replica-noflowcontrol
       - single-replica
       - single-replica-classic-query-engine
+      - single-replica-heuristic-bonsai
+      - single-replica-sampling-bonsai
```

Comment:
Thanks Milena for adding the sampling variant! Could you please also run a Replica Sampling variant to check that this workload is good to run?